### ISSUE TITLE
[HELP] - I can't install in minicoda  packages are not available

### ISSUE THREAD

**Issue by LuisMFC** [eqcorrscan/EQcorrscan#549]
# What do you need help with?

I can't install in minicoda. I search within conda eqcorrscan but it is not available.
Thank you.
LuisFC

# Provide an example so that we can reproduce your problem
1- I try to install
****************screen copy*****************************
```bash
(base) C:\Windows\system32>conda install conda-forge eqcorrscan
Collecting package metadata (current_repodata.json): done
Solving environment: unsuccessful initial attempt using frozen solve. Retrying with flexible solve.
Collecting package metadata (repodata.json): done
Solving environment: unsuccessful initial attempt using frozen solve. Retrying with flexible solve.

PackagesNotFoundError: The following packages are not available from current channels:

  - conda-forge
  - eqcorrscan

Current channels:

  - https://repo.anaconda.com/pkgs/main/win-64
  - https://repo.anaconda.com/pkgs/main/noarch
  - https://repo.anaconda.com/pkgs/r/win-64
  - https://repo.anaconda.com/pkgs/r/noarch
  - https://repo.anaconda.com/pkgs/msys2/win-64
  - https://repo.anaconda.com/pkgs/msys2/noarch

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.
```
**********screen end*************************

I'm search if it exists within the conda packages

**********screen copy*************************
```bash
(base) C:\Windows\system32>conda search eqcorrscan
Loading channels: done
No match found for: eqcorrscan. Search: *eqcorrscan*

PackagesNotFoundError: The following packages are not available from current channels:

  - eqcorrscan

Current channels:

  - https://repo.anaconda.com/pkgs/main/win-64
  - https://repo.anaconda.com/pkgs/main/noarch
  - https://repo.anaconda.com/pkgs/r/win-64
  - https://repo.anaconda.com/pkgs/r/noarch
  - https://repo.anaconda.com/pkgs/msys2/win-64
  - https://repo.anaconda.com/pkgs/msys2/noarch

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    -https://anaconda.org
```
*************screen end***************

# What help would you like?

I wish to install eqcorrscan. Know if there is another way to install to use in the Jupyter notebook environment

# What is your setup? (please complete the following information):
 - Operating System:
 Windows 10 Pro
 
 - Python version:
 ```bash
(base) C:\Windows\system32>conda info

     active environment : base
    active env location : C:\ProgramData\miniconda3
            shell level : 1
       user config file : C:\Users\LuisFC\.condarc
 populated config files : C:\Users\LuisFC\.condarc
          conda version : 23.7.4
    conda-build version : not installed
         python version : 3.11.4.final.0
       virtual packages : __archspec=1=x86_64
                          __cuda=11.1=0
                          __win=0=0
       base environment : C:\ProgramData\miniconda3  (writable)
      conda av data dir : C:\ProgramData\miniconda3\etc\conda
  conda av metadata url : None
           channel URLs : https://repo.anaconda.com/pkgs/main/win-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/win-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://repo.anaconda.com/pkgs/msys2/win-64
                          https://repo.anaconda.com/pkgs/msys2/noarch
          package cache : C:\ProgramData\miniconda3\pkgs
                          C:\Users\LuisFC\.conda\pkgs
                          C:\Users\LuisFC\AppData\Local\conda\conda\pkgs
       envs directories : C:\ProgramData\miniconda3\envs
                          C:\Users\LuisFC\.conda\envs
                          C:\Users\LuisFC\AppData\Local\conda\conda\envs
               platform : win-64
             user-agent : conda/23.7.4 requests/2.29.0 CPython/3.11.4 Windows/10 Windows/10.0.19044
          administrator : True
             netrc file : None
           offline mode : False
```
 - EQcorrscan version:
 
impossible to know

**Comment by d-chambers** [comment]
Did you remember to include conda-forge as the channel?

`conda install -c conda-forge eqcorrscan`

**Comment by LuisMFC** [comment]
Hi, yes I use:

conda install -c conda-forge eqcorrscan
conda install eqcorrscan
conda install conda-forge eqcorrscan

But maybe the problem is because I use 
Windows 10 Enterprice LTSC (sorry now I read that I put Win 10 pro)
I have installed other packages and had no problems

I will keep trying since the package is hosted

(base) C:\Windows\system32>conda search eqcorrscan
Loading channels: done
# Name                       Version           Build  Channel
eqcorrscan                     0.2.4     np111py35_1  conda-forge
eqcorrscan                     0.2.4     np111py36_1  conda-forge
eqcorrscan                     0.2.4     np112py35_1  conda-forge
eqcorrscan                     0.2.4     np112py36_1  conda-forge
..............................................................................................
..............................................................................................
eqcorrscan                     0.4.4 py310ha9ca8c9_2  conda-forge
eqcorrscan                     0.4.4 py310ha9ca8c9_3  conda-forge
eqcorrscan                     0.4.4 py310hbdeb650_0  conda-forge
eqcorrscan                     0.4.4 py310hbdeb650_1  conda-forge
eqcorrscan                     0.4.4  py37hdb909a5_0  conda-forge
eqcorrscan                     0.4.4  py37hdb909a5_1  conda-forge
eqcorrscan                     0.4.4  py39h430ef6e_2  conda-forge
eqcorrscan                     0.4.4  py39h430ef6e_3  conda-forge
eqcorrscan                     0.4.4  py39hb1d0a4c_0  conda-forge
eqcorrscan                     0.4.4  py39hb1d0a4c_1  conda-forge

**Comment by calum-chamberlain** [comment]
Kia ora @LuisMFC - the error you first posted was because you were missing the "`-c`" before `conda-forge`. As @d-chambers said, you need to use the command:
`conda install -c conda-forge eqcorrscan`
As you said that that did not work for you already, could you post the error that comes from that command please, then I can try and find out what is going on.

It also looks like you are installing eqcorrscan into your base conda environment, this is strongly *not* recommended. You should follow the installation instructions on the [website](https://eqcorrscan.readthedocs.io/en/latest/installation.html) and create an environment first and install into that. It is likely that the issue you are having is stemming from a mismatch between the python version in your base environment (which looks like 3.11) and the builds I have made for EQcorrscan, (which only go up to 3.9 because I haven't had time to keep up with Python updates).

# TLDR

Install EQcorrscan following the instructions on the [website](https://eqcorrscan.readthedocs.io/en/latest/installation.html#eqcorrscan-installation):
```bash
conda create -n eqcorrscan -c conda-forge eqcorrscan
```

**Comment by LuisMFC** [comment]
So as not to extend the topic.
CONCLUSION
Yes, I can install by doing this:
(creating a specific environment for the package)

conda create -n eqcorrscan -c conda-forge eqcorrscan
source activate eqcorrscan

(this works OK)



since I want to use eqcorrscan in the same environment as other packages.
No, I can install with just

conda install -c conda-forge eqcorrscan


That is the result if I want to install without creating the specific environment for eqcorrscan

(base) C:\Users\LuisFC>conda install -c conda-forge eqcorrscan
Collecting package metadata (current_repodata.json): done
Solving environment: unsuccessful initial attempt using frozen solve. Retrying with flexible solve.
Solving environment: unsuccessful attempt using repodata from current_repodata.json, retrying with next repodata source.
ResolvePackageNotFound:
   - conda==23.5.2

But I use in the env eqcorrscan

Thaks for all.

**Comment by calum-chamberlain** [comment]
You should be able to install EQcorrscan in another environment with other software, but
1. You should **not install software into the conda base environment** as a [general rule](https://docs.conda.io/projects/conda/en/latest/user-guide/getting-started.html#managing-environments)
2. As you saw in your search, EQcorrscan is only built for python versions up to 3.9. If your environment has a python version > 3.9 you won't be able to install eqcorrscan.
